src/pages/en/guides/styling.md

```diff
@@ -2,16 +2,22 @@
 layout: ~/layouts/MainLayout.astro
 title: Styling & CSS
 description: Learn how to style components with Astro.
+setup: |
+  import Since from '../../../components/Since.astro';
 ---
+Astro includes special handling to make writing CSS as easy as possible. You can use your favorite libraries, write your own css files, or take advantage of Astro's own scoped styling.
 
-Astro includes special handling to make writing CSS as easy as possible. Styling inside of Astro components is done by adding a `<style>` tag anywhere.
+## Astro scoped `<style>`
```

Comment:
This feels a bit like getting pushed into the deep end. Maybe a section before this that just re-introduces the concept and syntax of styling in Astro components?

- Styling Your Astro Components
- Scoped Styles
  - ...
- Global Styles
  - ...